test/__file_snapshots__/node--optional-chaining-0

```diff
@@ -1,12 +1,14 @@
 "use strict";
 
+var _obj$foo, _obj$foo$bar, _obj$qux;
+
 let obj = {
   foo: {
     bar: {
       baz: 42
     }
   }
 };
-let baz = obj?.foo?.bar?.baz; // 42
+let baz = obj === null || obj === void 0 ? void 0 : (_obj$foo = obj.foo) === null || _obj$foo === void 0 ? void 0 : (_obj$foo$bar = _obj$foo.bar) === null || _obj$foo$bar === void 0 ? void 0 : _obj$foo$bar.baz; // 42
 
-let safe = obj?.qux?.baz; // undefined
\ No newline at end of file
+let safe = obj === null || obj === void 0 ? void 0 : (_obj$qux = obj.qux) === null || _obj$qux === void 0 ? void 0 : _obj$qux.baz; // undefined
```

Comment:
Regression?

Reply:
That's the question, yes basically it shouldn't...

Reply:
So it's a bug in Chrome (hence node), forcing Babel to transform it. 

See 
https://github.com/babel/babel/issues/13145  
https://github.com/babel/babel/issues/13001

Reply:
Thanks; can we try `bugfixes: true`? https://github.com/babel/babel/issues/13145#issuecomment-818612268

Reply:
Already the case no? https://github.com/Gandi/babel-preset-gandi/pull/30/files#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R35

Reply:
@ChrisRast, @pascalduez: I don't know, do we match the condition for node? `bugfixes: options.targets && options.targets.esmodules === true,`

Reply:
I tried, no difference. And it's not something we do in the preset anyway.